test/common/testlib.py

```diff
@@ -1688,18 +1688,18 @@ def terminate_sessions() -> None:
             self.machine.execute("systemctl stop systemd-logind")
 
             # Wait for sessions to be gone
-            sessions = self.machine.execute("loginctl --no-legend list-sessions | awk '/web console/ { print $1 }'").strip().split()
-            for s in sessions:
+            sessions = self.machine.execute("loginctl --no-legend list-sessions | awk '/web console/ { print $1,$2 }'").strip().split()
+            for session in sessions:
+                sessionid, uid = session.split(',')
                 try:
-                    m.execute(f"while loginctl show-session {s}; do sleep 0.2; done", timeout=30)
+                    m.execute(f"while loginctl show-session {sessionid}; do sleep 0.2; done", timeout=30)
+                    # Terminate systemd user service
+                    m.execute(f"systemctl stop user@{uid}.service")
                 except RuntimeError:
                     # show the status in debug logs, to see what's wrong
-                    m.execute(f"loginctl session-status {s}; systemd-cgls", stdout=None)
+                    m.execute(f"loginctl session-status {sessionid}; systemd-cgls", stdout=None)
                     raise
 
-            # terminate all systemd user services for users who are not logged in
-            self.machine.execute("systemctl stop user@*.service")
```

Comment:
This change changes behaviour, as it used to stop non `web console` sessions. Is that wanted?

Reply:
In *theory*, `loginctl kill-user` should do that as well`, but we don't do that. Let's see what the tests say -- if it goes green, then ok. It may also only affect e.g. c-podman, but we can put it back if needed. This cleanup is unfortunately incredibly finnicky and brittle :cry:

Reply:
Yeah, you get errors like [this](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20400-d53d53ef-20240429-140734-rhel-9-5-other/TestAccounts-testBasic-rhel-9-5-127.0.0.2-2401-FAIL-1.png) from stale sessions :cry:

Reply:
So hmm should we generalize this? Or add another for loop to iterate over "normal sessions" except where `uid == 0`

Reply:
Hmm, I just saw that we already do the `loginctl kill-user` thing in `terminate_sessions()`. So I don't know, perhaps reproduce one of the failures locally and see how you can clean up after e.g. the "anton" test?